recipes/ringdecomposerlib/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "RingDecomposerLib" %}
+{% set version = "1.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rareylab/RingDecomposerLib/archive/refs/tags/v{{ version }}_rdkit.tar.gz
+  sha256: 944b5816712a48bbf88aa25d4300ce11871ddf6e971218eac08f90ed2192f715
+  patches:
+    - language_c.patch
+
+build:
+  number: 0
+
+  run_exports:
+    - {{ pin_subpackage(name|lower, max_pin='x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake
+    - ninja
+
```

Comment:
```suggestion

# NOTE: Python wrapper not built because it statically links to library,
# so the build steps needs patching. (static linking not allowed on conda-forge)

```
It looks like the python wrapper statically links to the library, which is not allowed. I think it's OK to wait for someone who wants the python wrapper to request it. Building the python wrapper with require some patching.


https://github.com/rareylab/RingDecomposerLib/blob/3a7ff93de0d9c4f6a5661508549c6063573f39c7/src/python/CMakeLists.txt#L21